docs/requirements.txt

```diff
@@ -0,0 +1,6 @@
+mkdocs == 1.4.3
+mkdocs-material == 9.1.12
+mkdocs-include-markdown-plugin == 4.0.4
+mkdocs-same-dir == 0.1.2
+mkdocs-exclude == 1.0.2
+mkdocs_exclude_search == 0.6.5
```

Comment:
Is there a reason why you pinned the versions? If not and it's working without pinning, please remove

Reply:
Alright, seems to be working without the pinning just as well. :)